docs/self-hosted-appcircle/install-server/cloud-providers/_configure-server.mdx

```diff
@@ -0,0 +1,36 @@
+After you successfully connect to the Appcircle instance, the first thing you should do is start a system update. Although the Appcircle AMI is up-to-date, it is recommended that you perform security updates again.
+
+```bash
+sudo apt update && \
+sudo apt upgrade
+```
+
+The Appcircle server directory is located in the `$HOME` directory as the `appcircle-server` folder.
+
+```bash
+ls -l "$HOME"
+```
+
+Change the current working directory to that folder.
+
+```bash
+cd "$HOME/appcircle-server"
+```
+
+:::info
+
+**If you are a licensed user**, please [contact us](https://appcircle.io/support/) to get the licensed Appcircle zip package. You should [upgrade](/self-hosted-appcircle/update) the pre-installed package in the instance.
+
+Also, put the `cred.json` file you received from us into the `appcircle-server` folder.
+
+Please [contact us](https://appcircle.io/support/) to purchase an enterprise license if you don't have one.
+
+:::
+
+Now you're ready to configure the Appcircle server according to your needs. Follow the detailed [configuration](/self-hosted-appcircle/install-server/docker#3-configure) steps.
+
+You should also configure the [DNS](/self-hosted-appcircle/install-server/docker#4-dns-settings) settings for your Appcircle server instance. Create `A` and `CNAME` records for your instance.
+
+After the configuration is done, [initialize the project](/docs/self-hosted-appcircle/install-server/docker.md#5-initialize-vault), [run the server](/self-hosted-appcircle/install-server/docker#6-run-server), and go to the Appcircle server dashboard using the main [domain](/self-hosted-appcircle/install-server/docker#4-dns-settings) you defined.
```

Comment:
```suggestion
Now you're ready to configure the Appcircle server according to your needs. 

1. Follow the [server configuration](/self-hosted-appcircle/install-server/docker#3-configure) steps.

2. Configure the [DNS settings](/self-hosted-appcircle/install-server/docker#4-dns-settings) for your Appcircle server instance. Create `A` and `CNAME` records for your instance.

3. [Initialize the project](/docs/self-hosted-appcircle/install-server/docker.md#5-initialize-vault)
4. [Run the server](/self-hosted-appcircle/install-server/docker#6-run-server)

Finally you can go to the Appcircle server dashboard using the main [domain](/self-hosted-appcircle/install-server/docker#4-dns-settings) you defined.
```

Giving these steps as list may improve the readability.

Reply:
We can handle this on another PR. Other pages should get updated too.

Reply:
Why not accept it if the content is true? It already seems to be suggested for a common component (mdx) that's shared.

Reply:
I was planning to use this as common component but the "AWS tip" spoils the plan. We should handle separately for each page "AWS" "Azure" "Podman" "Docker". Will be handled in this PR.

Reply:
> I was planning to use this as common component but the "AWS tip" spoils the plan. 

Hımm. After your comment, I checked the whole component.

> ... We should handle separately for each page

Yes. For instance, this component has a concept `AMI` at the top but it's not a valid concept for Azure.